.github/workflows/codegenerator.yml

```diff
@@ -0,0 +1,153 @@
+on:
```

Comment:
uber-nit: codegenerator -> code_generator.yml

Reply:
done in 10430e1